.circleci/continue_config.yml

```diff
@@ -0,0 +1,235 @@
+version: 2.1
+
+orbs:
+  codecov: codecov/codecov@3.2.4
+
+parameters:
+  classification:
+    type: boolean
+    default: false
+  detection:
+    type: boolean
+    default: false
+  fr:
+    type: boolean
+    default: false
+  generic:
+    type: boolean
+    default: false
+  misc:
+    type: boolean
+    default: false
```

Comment:
where do we use this parameter?

Reply:
Path-filter will output the parameters into a json file. It is then used at step `Generate Config` to manipulate the continued yaml.

Reply:
🤔 i didn't see it in the mapping, maybe i misunderstand something

Reply:
Understood your question now, will follow up.